lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -151,15 +143,15 @@ def __init__(self, name: AnyStr, convert: Callable[[AnyStr], Any]):
         self.friendly = name
         self.convert = convert
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.name
 
-    def convert(self, value):
+    def convert(self, value: Any) -> Any:
```

Comment:
The implementation of this function appears to call itself...bottomlessly.

Reply:
OK, that's weird. Somehow I never even noticed that I'd used the same name, and I have no idea how this worked.

Maybe my laptop just runs the infinite recursion really quickly ... ;-)